mitxpro/settings.py

```diff
@@ -599,6 +600,8 @@
 )
 CELERY_TASK_ALWAYS_EAGER = get_bool("CELERY_TASK_ALWAYS_EAGER", False, dev_only=True)
 CELERY_TASK_EAGER_PROPAGATES = get_bool("CELERY_TASK_EAGER_PROPAGATES", True)
+CRON_COURSE_CERTIFICATES_HOURS = get_string("CRON_COURSE_CERTIFICATES_HOURS", 0)
+CRON_COURSE_CERTIFICATES_DAYS = get_string("CRON_COURSE_CERTIFICATES_DAYS", None)
```

Comment:
`get_string("CRON_COURSE_CERTIFICATES_DAYS", '*')` simplifies the usage later on by avoiding the requirement of `or '*'`